lit-cli/setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+import platform
+
+requirements = ['Click']
+
+# Enable Cu90 only when using linux with cuda enabled
```

Comment:
Why do you need GPU Support now? Its not supported, we can bring it in when there is support. don't want to add untested code